content/en/docs/started/kubeflow-examples.md

```diff
@@ -5,9 +5,12 @@ weight = 30
                     
 +++
 
-This section introduces the examples in the 
+{{% alert title="Warning" color="warning" %}}
+This section introduces some examples in the 
 [kubeflow/examples](https://github.com/kubeflow/examples) repository.
-Before using a sample, check the sample's README file for known issues.
+There are out-of-date examples in this repository, check the sample's README file before using.
```

Comment:
Perhaps we could say something more positive, like:

```
{{% alert title="Warning" color="warning" %}}
Many of the examples in [kubeflow/examples](https://github.com/kubeflow/examples) have not been tested with newer versions of Kubeflow, and some may not work correctly. 
Please refer to the README of your chosen example.
{{% /alert %}}
```

Reply:
@zijianjoy @thesuperzapper
 https://docs.google.com/spreadsheets/d/1cNL-YtTpvBIfLffvPfn9Fz-737pkJNj7uUGHU5DQvu0/
there are only 2 examples use KF v1.3+
https://github.com/kubeflow/examples/blob/master/Natural-Language-Processing
https://github.com/kubeflow/examples/tree/master/FaceNet-distributed-training

Should we Add these ?

Reply:
@zijianjoy @thesuperzapper  Also I will update or make a new https://github.com/kubeflow/examples/tree/master/test/ It would automate the current testing process and make and trigger it to test every example after a release and notify the owners if it fails with the latest release there is currently no automated testing in https://github.com/kubeflow/examples

can @jlewi @zijianjoy @thesuperzapper we make an e2e testing proposal like this
https://docs.google.com/document/d/1iaVTjr9XVQYb1_kdpq__vq9S9hHaDrX31LWqRs3aujc/

Reply:
It makes sense for having positive tone, applying the suggestion with some adjustment.